Extensions/mute.js

```diff
@@ -566,53 +563,54 @@ XKit.extensions.mute = new Object({
+		// var m_link_class = "";
+		// var m_chat_class = "";
+		// var m_audio_class = "";
+		// var m_video_class = "";
+		// var m_asks_class = "";
 		var m_reblogs_class = "";
 		var m_originals_class = "";
 
-		var user_object = XKit.extensions.mute.return_user_object(ud);
-
-		if (user_object !== -1) {
-			if (user_object.regular === true) { m_regular_class = "selected"; }
-			if (user_object.photo === true) { m_photo_class = "selected"; }
-			if (user_object.quote === true) { m_quote_class = "selected"; }
-			if (user_object.link === true) { m_link_class = "selected"; }
-			if (user_object.chat === true) { m_chat_class = "selected"; }
-			if (user_object.audio === true) { m_audio_class = "selected"; }
-			if (user_object.video === true) { m_video_class = "selected"; }
-			if (user_object.asks === true) { m_asks_class = "selected"; }
-			if (typeof user_object.reblogs !== "undefined") {
-				if (user_object.reblogs === true) { m_reblogs_class = "selected"; }
-			}
-			if (typeof user_object.originals !== "undefined") {
-				if (user_object.originals === true) { m_originals_class = "selected"; }
-			}
-		}
-
-
-		var m_html =	"<div class=\"xkit-mute-options\">" +
-					"<div data-type=\"regular\" class=\"xkit-mute-option regular " + m_regular_class + "\">&nbsp;</div>" +
-					"<div data-type=\"asks\" class=\"xkit-mute-option asks " + m_asks_class + "\">&nbsp;</div>" +
-					"<div data-type=\"photo\" class=\"xkit-mute-option photo " + m_photo_class + "\">&nbsp;</div>" +
-					"<div data-type=\"quote\" class=\"xkit-mute-option quote " + m_quote_class + "\">&nbsp;</div>" +
-					"<div data-type=\"link\" class=\"xkit-mute-option link " + m_link_class + "\">&nbsp;</div>" +
-					"<div data-type=\"chat\" class=\"xkit-mute-option chat " + m_chat_class + "\">&nbsp;</div>" +
-					"<div data-type=\"audio\" class=\"xkit-mute-option audio " + m_audio_class + "\">&nbsp;</div>" +
-					"<div data-type=\"video\" class=\"xkit-mute-option video " + m_video_class + "\">&nbsp;</div>" +
-				"</div>";
+		// var user_object = XKit.extensions.mute.return_user_object(ud);
+
+		// if (user_object !== -1) {
+		// 	if (user_object.regular === true) { m_regular_class = "selected"; }
+		// 	if (user_object.photo === true) { m_photo_class = "selected"; }
+		// 	if (user_object.quote === true) { m_quote_class = "selected"; }
+		// 	if (user_object.link === true) { m_link_class = "selected"; }
+		// 	if (user_object.chat === true) { m_chat_class = "selected"; }
+		// 	if (user_object.audio === true) { m_audio_class = "selected"; }
+		// 	if (user_object.video === true) { m_video_class = "selected"; }
+		// 	if (user_object.asks === true) { m_asks_class = "selected"; }
+		// 	if (typeof user_object.reblogs !== "undefined") {
+		// 		if (user_object.reblogs === true) { m_reblogs_class = "selected"; }
+		// 	}
+		// 	if (typeof user_object.originals !== "undefined") {
+		// 		if (user_object.originals === true) { m_originals_class = "selected"; }
+		// 	}
+		// }
```

Comment:
It looks like the above section might still be needed in order to update the option checkboxes to the current settings!

Reply:
Thanks! Good eye; should be fixed now.